xml/security_firewall.xml

```diff
@@ -965,13 +965,35 @@ nfs-rpc
     <title>nftables as the default backend </title>
     <para>
     The default backend for &firewalld; is nftables. nftables is a framework by the Linux <systemitem>netfilter</systemitem> project
-  that provides packet filtering, network address translation (NAT) and other packet mangling. nftables is a replacement for
-  iptables, ip6tables, arptables, ebtables, and ipset.
+  that provides packet filtering, network address translation (NAT) and other packet mangling. nftables reuses the existing Netfilter
+  subsystems such as the connection tracking system, userspace queueing,logging and the hook infrastructure.
+  nftables is a replacement for iptables, ip6tables, arptables, ebtables, and ipset.
 </para>
     <para>
-  There are many advantages of using nftables such as, faster rule updates, built-in sets and a single framework
-  for IPv4 and IPv6 protocols.
+  The advantages of using nftables include:
     </para>
+    <itemizedlist mark="bullet" spacing="normal">
+      <listitem>
+       <para>
+
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        One framework for both the IPv4 and IPv6 protocols
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        Rules are applied automatically instead of fetching, updating and storing a complete rule set.
+        </para>
+      </listitem>
+      <listitem>
+       <para>
+        Monitor trace events in the <emphasis>nft</emphasis> tool and debugging and tracing in the rule set <emphasis>nftrace</emphasis>.
```

Comment:
The second part of the sentence doesn't look very sensical. How about:

`and debug and trace the rule set via <emphasis>nftrace</emphasis>`

Reply:
Yes , this is much better , implemented :)